docs/PLUGINS.md

```diff
@@ -277,24 +277,16 @@ You also can write a test case to check export from a live instance is well supp
     @Test
     @ConfiguredWithCode("configuration-as-code.yml")
     public void export_configuration() throws Exception {
-        YourGlobalConfiguration globalConfiguration = YourGlobalConfiguration.get();
-
         ConfiguratorRegistry registry = ConfiguratorRegistry.get();
         ConfigurationContext context = new ConfigurationContext(registry);
-        final Configurator c = context.lookupOrFail(YourGlobalConfiguration.class);
-
-        @SuppressWarnings("unchecked")
-        CNode node = c.describe(globalConfiguration, context);
-        assertNotNull(node);
-        final Mapping mapping = node.asMapping();
-
-        assertEquals(mapping.getScalarValue("expected-property"), "expected-value");
-        assertEquals(mapping.getScalarValue("another-property"), "another-value");
-        
-        List<CNode> yourList = mapping.get("a-list-property").asSequence();
-        assertEquals("list size", 5, yourList.size());
-        assertEquals(yourList.get(0).getScalar().getValue(), "expected-value");
-        assertEquals(yourList.get(0).asMapping().getScalarValue("key"), "expected-key");
+        final CNode yourAttribute = getUnclassifiedRoot(context).get("<your-attribute>");
+
+         String exported = toYamlString(yourAttribute);
+
+         String expected = new String(Files.readAllBytes(Paths.get(getClass()
```

Comment:
Perhaps provide another util method?
Assumes it should use `getClass().getResource()` unless String path is provided?

Reply:
Actually, the configured as code annotation could use the very same method :laughing:

Reply:
not sure what you mean?
There's Resources.toString("") in guava which could be used instead but wasn't 100% sure if guava was everywhere

Reply:
Just no to guava :) I'll submit my suggestion during the weekend.

Reply:
What I meant I am Saying create a method for `toStringFromYamlFile` to shorten the whole `Files.readAllBytes(Paths...(getClass()...)` nonsense

Reply:
I can't find a way that works in a static method in another project without using guava or spring

The classloaders aren't correct...

Reply:
You have access to Junit methods that should take care of the same.

Reply:
I can't see any special junit methods for it?
The ConfiguredWithCodeRule has access to the class in the users test so can use the correct class loader

Reply:
`Thread.currentThread().getStackTrace()[2].getClass`?
Rest should work

Reply:
that doesn't work in tests in jcasc...

Reply:
`toStringFromYamlFile(this, "my.yml")`?

Reply:
done that 👍